ansible/roles/software/ckan/catalog/ckan-app/templates/catalog-next/etc_ckan_production.ini.j2

```diff
@@ -0,0 +1,243 @@
+#
+# CKAN - Pylons configuration
+#
+# These are some of the configuration options available for your CKAN
+# instance. Check the documentation in 'doc/configuration.rst' or at the
+# following URL for a description of what they do and the full list of
+# available options:
+#
+# http://docs.ckan.org/en/latest/maintaining/configuration.html
+#
+# The %(here)s variable will be replaced with the parent directory of this file
+#
+
+[DEFAULT]
+
+# WARNING: *THIS SETTING MUST BE SET TO FALSE ON A PRODUCTION ENVIRONMENT*
+debug = false
+
+[server:main]
+use = egg:Paste#http
+host = 0.0.0.0
+port = 5000
+
+[app:main]
+use = egg:ckan
+full_stack = true
+cache_dir = /tmp/%(ckan.site_id)s/
+beaker.session.key = ckan
+
+# CKAN caching
+ckan.cache_expires = 300
+
+# This is the secret token that the beaker library uses to hash the cookie sent
+# to the client. `paster make-config` generates a unique value for this each
+# time it generates a config file.
+beaker.session.secret = {{ ckan_instance_secret }}
+
+beaker.session.type={{ ckan_session_store_type | default('ext:database') }}
+beaker.session.cookie_expires=true
+beaker.session.url=%(sqlalchemy.url)s
+beaker.session.timeout=3000
+beaker.session.lock_dir=/var/tmp/ckan/lock
+
+# `paster make-config` generates a unique value for this each time it generates
+# a config file.
+app_instance_uuid = {{ ckan_instance_uuid }}
+
+# repoze.who config
+who.config_file = %(here)s/who.ini
+who.log_level = warning
+who.log_file = %(cache_dir)s/who_log.ini
+# Session timeout (user logged out after period of inactivity, in seconds).
+# Inactive by default, so the session doesn't expire.
+who.timeout = 900
+# who.httponly = True # DEFAULT True already
+who.secure = {{ who_secure }}
+
+## Database Settings
+db_user = {{ catalog_db_user }}
+db_pass = {{ catalog_db_pass }}
+db_server = {{ catalog_ckan_db_host }}
+db_database = {{ catalog_db_name }}
+sqlalchemy.url = postgresql://%(db_user)s:%(db_pass)s@%(db_server)s/%(db_database)s
+
+
+## Site Settings
+
+ckan.site_url = {{ ckan_site_domain }}
+
+#ckan.use_pylons_response_cleanup_middleware = true
+
+## Authorization Settings
+
+ckan.auth.anon_create_dataset = false
+ckan.auth.create_unowned_dataset = false
+ckan.auth.create_dataset_if_not_in_organization = false
+ckan.auth.user_create_groups = false
+ckan.auth.user_create_organizations = false
+ckan.auth.user_delete_groups = false
+ckan.auth.user_delete_organizations = false
+ckan.auth.create_user_via_api = false
+ckan.auth.create_user_via_web = true
+ckan.auth.roles_that_cascade_to_sub_groups = admin
+
+
+## Search Settings
+
+ckan.site_id = geo.gov
+
+solr_server = {{ catalog_ckan_solr_host }}
+solr_url = http://%(solr_server)s:{{ catalog_ckan_solr_port }}/solr/catalog-next
```

Comment:
The solr core name should probably be some thing more like catalog-28... the catalog-next name will go away once we ship, but the core will live on.

Reply:
Ah.  So, I should update  GSA/datagov-deploy-solr and change the name?

Reply:
Not as part of this PR. We'll def want to get this right for staging and production, but I would merge as is.